projects/packages/blaze/src/class-dashboard-rest-controller.php

```diff
@@ -853,13 +859,14 @@ protected function get_wp_error( $response_body, $response_code = 500 ) {
 			}
 		}
 
-		$error_message = isset( $response_body['message'] ) ? $response_body['message'] : 'unknown remote error';
+		$response_body['code']   = $error_code;
+		$response_body['status'] = $response_code;
 
-		return new WP_Error(
-			$error_code,
-			$error_message,
-			array( 'status' => $response_code )
-		);
+		if ( ! isset( $response_body['errorMessage'] ) ) {
+			$response_body['errorMessage'] = 'Unknown remote error';
+		}
+
+		return new \WP_REST_Response( $response_body, $response_code );
```

Comment:
Tiny nitpick, but I think things would be a bit more readable that way:

```diff
diff --git a/projects/packages/blaze/src/class-dashboard-rest-controller.php b/projects/packages/blaze/src/class-dashboard-rest-controller.php
index ab79708fd8..49df7d8b0b 100644
--- a/projects/packages/blaze/src/class-dashboard-rest-controller.php
+++ b/projects/packages/blaze/src/class-dashboard-rest-controller.php
@@ -859,12 +859,9 @@ class Dashboard_REST_Controller {
 			}
 		}
 
-		$response_body['code']   = $error_code;
-		$response_body['status'] = $response_code;
-
-		if ( ! isset( $response_body['errorMessage'] ) ) {
-			$response_body['errorMessage'] = 'Unknown remote error';
-		}
+		$response_body['code']         = $error_code;
+		$response_body['status']       = $response_code;
+		$response_body['errorMessage'] = $response_body['errorMessage'] ?? 'Unknown remote error';
 
 		return new \WP_REST_Response( $response_body, $response_code );
 	}

```

Reply:
Thanks for the review @jeherve! I like your suggestion, it will make the code easier to read.
I applied the change 👍